Bottom: 39854363d58df4e30acb266d9bdb35dc64bf970c
Top:    39854363d58df4e30acb266d9bdb35dc64bf970c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-22 00:31:57 +0100

Try adding polygon objects after each clipping stage. rtree->faster??

*** ACTUALLY SLOWS STARTUP DOWN ***



---


